### DISCUSSION TITLE
SQS Queues and AWS Destinations

### DISCUSSION THREAD

**Discussion by ezmiller** [getlift/lift#224] [Constructs]
I have been using the SQS construct a bit. It configures Dead-Letter-Queues by default. It looks like the preferred solution is now to use AWS "Destinations" instead of DLQs in part because you can receive more context for the failure. Serverless itself supports these as of 3/2020 (see [here](https://www.serverless.com/blog/lambda-destinations/)). 

I'm curious if there's any support for this within lift. I didn't see anything in the docs. If not, is there a specific reason for that design choice?

**Comment by mnapoli** [comment]
Destinations don't work with SQS as it's an "event-source mapping" kind of invocation, not an async invocation.

**Comment by ezmiller** [comment]
I subsequently figured that out as well.  :)
